### PR TITLE
Host preflights for EC

### PR DESCRIPTION
**Pull request by paigecalvert**

### CODE REVIEW

File: docs/vendor/embedded-overview.mdx [hunk 1]
```diff
@@ -234,6 +232,37 @@ The following are never proxied:
 * Proxy settings are not automatically propagated to your [Helm extensions](/reference/embedded-config#helm-extensions) and must be manually configured.
 * Proxy settings cannot be changed after installation or during upgrade.
 
+### Host Preflight Checks
```
Comment (paigecalvert) [inline docs/vendor/embedded-overview.mdx:235]: Nested under the Install heading

File: docs/vendor/preflight-support-bundle-about.md [hunk 2]
```diff
@@ -71,6 +71,16 @@ For applications installed with KOTS, it is possible to block the deployment of
 
 You can add required preflight checks for your application by including `strict: true` for the target analyzer in the preflight specification. For more information, see [Block Installation with Required Preflights](preflight-defining#strict) in _Defining Preflight Checks_.
 
+### (KOTS Only) Host Preflights for VM or Bare Metal Installations {#host-preflights}
```
Comment (paigecalvert) [inline docs/vendor/preflight-support-bundle-about.md:74]: ^ Added this section about host preflights to the About Preflights and Support Bundles topic. Gives space to explain that host preflights are a thing for both kurl and EC, and space to link out to the related topics for more info

File: docs/vendor/embedded-overview.mdx [hunk 3]
```diff
@@ -234,6 +232,37 @@ The following are never proxied:
...
+
+During installation, Embedded Cluster automatically runs a default set of _host preflight checks_. The default host preflight checks are designed to verify that the installation environment meets the requirements for Embedded Cluster, such as:
+* The disk containing directory has sufficient space
```
Comment (ajp-io) [inline docs/vendor/embedded-overview.mdx:238]: The spec says `directory /`, but we're actually changing that to search a few different places instead of just `/`, so I'm going to reword this to be more general.
```suggestion
* The system has sufficient disk space
```
